index.bs

```diff
@@ -573,12 +574,16 @@ authorizing an authenticator with which to complete the operation.
     to locate it later for use, and also contains metadata that can be used by the <a>[RP]</a> to assess the strength of the
     credential during registration.
 
-    The <dfn>credential</dfn> attribute contains an object whose attributes state the type of, and identifier for, the 
-    credential represented by {{ScopedCredentialInfo}}.
+    The <dfn>clientData</dfn> member contains the <a>clientDataJSON</a> (see [[#signature-format]]) passed to the authenticator
+    by the client in order to generate this credential. The exact JSON encoding must be preserved as a cryptographic hash
+    (<a>clientDataHash</a>) has been computed over it.
 
-    The <dfn>attestation</dfn> attribute contains an attestation statement returned by the authenticator. This provides
-    information about the credential and the authenticator it is held in, such as the <a>credential public key</a> and the level
-    of security assurance provided by the authenticator.
+    The <dfn>attestationObject</dfn> element contains the <a>attestation object</a>. The contents of this object are determined
+    by the <a>attestation statement format</a> used by the authenticator. This object is opaque to, and cryptographically
+    protected against tampering by, the client. It contains the unique identifier of the credential, the <a>credential public
```

Comment:
AAGUID is the authenticator's identifier, not the credential's. I tweaked the wording along these lines though.